src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -181,14 +179,11 @@ def handler(ex):
     "Storage Queue Data Contributor"
     "Storage Queue Data Reader"
 
-If you want to use the old authentication method and allow querying for the right account key, please use the "--auth-mode" parameter and "key" value.
+If you want to use the old authentication method and allow querying for the right account key, please use the "--auth-mode" parameter and "key" value. ErrorCode: AuthenticationFailed
                 """
-                logger.error(message)
-                return
+                ex.args = (message,)
             if isinstance(ex, t_error) and ex.status_code == 409 and ex.error_code == 'NoPendingCopyOperation':
-                logger.error(ex.args[0])
-                return
-            raise ex
```

Comment:
Can you tell me why do you remove `raise ex` here? In this way, how do you handle other exceptions?

Reply:
Other exceptions are raised in the outer layer of the call stack.

Reply:
During run-time, `first` is `show_exception_handler`, which raises exception anyway. 

https://github.com/Azure/azure-cli/blob/e47ee99ea2ff1476d3943d81f0403b4dcc638221/src/azure-cli-core/azure/cli/core/commands/arm.py#L754-L761